error: call to undeclared function 'wrap_report'

Hi, 

I defined the DUDECT_TRACE switch and got the `dudect.h:392:3: error: call to undeclared function 'wrap_report'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]` error.
The fonction is not defined in dudect.h

--- a/src/dudect.h
+++ b/src/dudect.h
@@ -346,11 +346,11 @@ static dudect_state_t report(dudect_ctx_t *ctx) {
   }
 
   printf("t-test on raw measurements\n");
-  wrap_report(ctx->ttest_ctxs[0]);
+  report_test(ctx->ttest_ctxs[0]);
 
   printf("t-test on cropped measurements\n");
   for (size_t i = 0; i < DUDECT_NUMBER_PERCENTILES; i++) {
-    wrap_report(ctx->ttest_ctxs[i + 1]);
+    report_test(ctx->ttest_ctxs[i + 1]);
   }
 
   printf("t-test for second order leakage\n");

